tensorboard/webapp/metrics/views/right_pane/settings_view_component.scss

```diff
@@ -34,7 +34,10 @@ section {
   margin: 0 0 12px 0;
 }
 
-section .control-row:not(:last-child) {
+// Add additional margin between most control elements. The exception is when
+// the second control row is a checkbox, in which case we just rely on the
+// padding that is already included in the checkbox.
+section .control-row:not(:has(+ .control-row > mat-checkbox)):not(:last-child) {
```

Comment:
😆 this is a wild selector. I can't figure out how to write this as a single sentence without being a runon.
A `.control-row` that is inside of a `section` and comes immediately after another `.control-row` which contains a `mat-checkbox` and is not the `:last-child`.